### ISSUE TITLE
Trouble running as standalone

### ISSUE THREAD

**Issue by youjinp** [SwiftGen/SwiftGen#863]
# Goal
Executing a standalone binary in a project.

# Problem
I ran the commands specified in the doc, namely `bundle exec rake cli:install`, and everything compiled fine.

But after copying the `bin` folder to another location and deleting the original directory, running `swiftgen` gives this error: 

```
SwiftGenCLI/resource_bundle_accessor.swift:11: Fatal error: could not load resource bundle: from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/SwiftGen_SwiftGenCLI.bundle or <compile-path>/.build/arm64-apple-macosx/release/SwiftGen_SwiftGenCLI.bundle
```

Seems like some path is hardcoded?

# Also tried
```
bundle exec rake release:zip
```

**Comment by youjinp** [comment]
I should mention that I did this on the latest commit in the develop branch

**Comment by djbe** [comment]
Weird, just tried it myself:
- `rake release:zip`
- copy the zip somewhere
- delete the old `.build/` folder
- extract zip
- run `swiftgen`, or `swiftgen --version`, or `swiftgen config run --config ...`, all of which worked

What macOS version are you on, and what Xcode version?

Note: I tried it both with Xcode 12.5 and Xcode 13.0.

**Comment by youjinp** [comment]
Just tried again with your steps just to make sure, and I got the same FatalError crash when running

- `swiftgen` (with valid `swiftgen.yml` file)
- `swiftgen config run --config <valid swiftgen.yml file>`.

macOS: Big Sur, 11.4
Mac: Mac mini (M1, 2020)
Xcode: both 12.5 and 13.0

**Comment by djbe** [comment]
I'm... honestly not sure what's going on 😕 The only difference is that you're on an M1 mac with 11.4, and I'm on an Intel mac with 11.6.

The best thing to avoid such machine differences is to use 1 machine 😆 our GitHub actions test the `release:zip` step, and attach the result to each run. You can find one here for example: https://github.com/SwiftGen/SwiftGen/actions/runs/1301022796
Download that zip and try to use that.

**Note**: You will have to remove the quarantine flag set by macOS, by for example doing "right-click > open" once, or using `xattr -r -d com.apple.quarantine swiftgen` (untested)

**Comment by youjinp** [comment]
Alright, I'll try it on an Intel Mac next time :) Just downloaded the zip from the action and it works beautifully. Thank you!

**Comment by djbe** [comment]
It should support M1 though, I still have no idea where it's going wrong there.

From some quick searches, seems like it's the same issue as https://github.com/vapor-community/heroku-buildpack/issues/63, where they state it's a [bug in Swift 5.4](https://bugs.swift.org/browse/SR-14555), with a fix [cherry picked to Swift 5.5](https://github.com/apple/swift-package-manager/pull/3467) and thus released in Xcode 13. So if you're using Xcode 13, it "should work" 🤷
